subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -2064,3 +2068,68 @@ void bt_mesh_cfg_cli_timeout_set(int32_t timeout)
 {
 	msg_timeout = timeout;
 }
+
+int bt_mesh_comp_p0_get(struct bt_mesh_comp_p0 *page,
+			struct net_buf_simple *buf)
+{
+	if (buf->len < 10) {
+		return -EINVAL;
+	}
```

Comment:
`CHECKIF()` here as well?

Reply:
I left it out on purpose, as it's doing a comparison on on-air data. With `CHECKIF`, it can turn into an assert or be skipped, which would force the application to do their own check. This would sort of negate the intended effect of this macro

Reply:
Right. Makes sense.